packages/client/src/redux/user.ts

```diff
@@ -13,7 +13,7 @@ export const userSlice = createSlice({
   initialState,
   reducers: {
     login: (state, { payload }) => {
-      state.me = payload.user;
+      state.me = payload;
     },
   },
```

Comment:
This might not need to change on the client side, instead passing { user: { ... } } as the response would work. Not sure which way to go with